oci/spec_opts_linux_test.go

```diff
@@ -106,3 +110,114 @@ func TestDropCaps(t *testing.T) {
 		}
 	}
 }
+
+func TestGetDevices(t *testing.T) {
+	dir, err := ioutil.TempDir("/dev", t.Name())
+	if err != nil {
+		if os.IsPermission(err) {
+			t.Skip("Test requires root")
```

Comment:
we can use `testutil.RequiresRoot` here. 

REF: https://pkg.go.dev/github.com/containerd/containerd@v1.4.4/pkg/testutil#RequiresRoot

Reply:
Yeah I had a reason for doing it this way but I can't recall why (maybe something related to passing the root flag while I was iterating was annoying?).

Reply:
Fixed.